src/app/pages/system/general-settings/localization-form2/components/ix-input/ix-input.component.html

```diff
@@ -0,0 +1,71 @@
+<div fxLayout="row wrap" fxLayoutGap="24px" fxLayoutAlign="start start" fxFlex="100%" fxFlex.gt-xs="calc({{'100%'}} - 16px)">
+  <div fxFlex="100%" fxFlex.gt-xs="calc({{'100%'}} - 16px)" [ngClass]="formInline ? 'form-inline' : 'form-line'">
+  <div id="{{name}}" class="dynamic-field form-input form-element" [ngClass]="fieldShow" [class.has-pwtoggle]="togglePw" [class.has-tooltip]="tooltip" *ngIf="!isHidden">
+    <div *ngIf="placeholder" class="label-container">
+      <label class="label input-text">
+        {{placeholder | translate}}
+        <span *ngIf="required">*</span>
+      </label>
+      <tooltip *ngIf="tooltip" [header]="placeholder" [message]="tooltip" [position]="tooltipPosition ? tooltipPosition : null"></tooltip>
+    </div>
+
+    <!-- need to prevent browser autofill :/ -->
+    <input name = "DummyUsername" type="text" style="display:none;">
+    <input name = "DummyPassword" type="password" style="display:none;">
+
+    <div class="full-width" [ngClass]="{'has_toggle_pw': togglePw}" *ngIf="inputType != 'file' else elseBlock">
+      <mat-spinner
+        diameter='30'
+        class="form-input-spinner"
+        id="{{name}}_form-input-spinner"
+        *ngIf="isLoading">
+      </mat-spinner>
+      <div *ngIf="maskValue">
+        <p>{{'Enter' | translate}} <strong>{{maskValue}}</strong> {{'below to confirm.' | translate}}</p>
+        <p [ngStyle]="{'position': 'relative', 'top':'15px', 'opacity': '.38'}">{{maskValue}}</p>
+      </div>
+      <div class="input-container">
+        <span *ngIf="searchable" class="searchable"><mat-icon>search</mat-icon></span>
+        <input [(ngModel)]="value" [ngStyle]="{'padding-left': searchable ? '2rem' : '', 'padding-right': shouldShowResetInput() ? '2rem': ''}" [type]="inputType ? inputType : 'text'"
```

Comment:
Two approaches are being mixed here – ngModel and reactive forms. We should just use reactive forms.